README.md

```diff
@@ -14,10 +14,11 @@ RowsX is a versatile Chrome extension designed to streamline web scraping tasks,
 
 ## Why Open-source RowsX?
 
-RowsX was born during a team hackathon in January of 2024. We built it to solve a problem we see our customers struggle with everyday: getting data from the web to a spreadsheet. Since launching it, more than two thousand people have used it to import lists of data from hundreds of sites, internal tools and back office systems.
-
 We're opening RowsX to the community and invite everyone to contribute with new features, support to new websites or new ideas to improve it. 
 
+RowsX was born during a team hackathon in January of 2024. We built it to solve a problem we see our customers struggle with everyday: getting data from the web to a spreadsheet. The Rows platform lets you import data from files, via Integrations with APIs and SaaS services, and conectors on 3rd party platforms that call our Rows API. RowsX extends the importing options to the Browser! Since launching it, more than two thousand people have used it to import lists of data from countless sites, internal tools and back office systems.
```

Comment:
in bold the changes:

RowsX was born during a team hackathon in January of 2024. We built it to solve a problem we see our customers struggle with `every day`: getting data from the web `into a spreadsheet`. The Rows platform lets you import data from files, via Integrations with APIs and SaaS services, and `connectors` on 3rd party platforms that call our Rows API. RowsX extends the importing options to the Browser! `Since its launch`, more than two thousand people have used it to import lists of data from countless sites, internal tools and back office systems.